README.md

```diff
@@ -1,6 +1,6 @@
 ![Plonky3-powered-by-polygon](https://github.com/Plonky3/Plonky3/assets/86010/7ec356ad-b0f3-4c4c-aa1d-3a151c1065e7)
 
-Plonky3 is a toolkit for implementing polynomial IOPs (PIOPs), such as PLONK and STARKs. It aims to support several polynomial commitment schemes, such as Brakedown.
```

Comment:
I think it should actually be Brakedown, unlikely the more common English word, see https://eprint.iacr.org/2021/1043